src/ui/profile_preferences.ui

```diff
@@ -769,6 +769,16 @@
             </property>
            </widget>
           </item>
+          <item row="2" column="0">
+           <widget class="QCheckBox" name="checkBox_echoLuaErrors">
+            <property name="toolTip">
+             <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Prints Lua errors to the main console.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
+            </property>
+            <property name="text">
+             <string>Echo Lua Errors</string>
```

Comment:
Whatever changes to the text of it you want, I can absolutely whip up. Just to be sure though, you're saying the label text should stay the same as "Echo Lua errors" or should I make that more informative too(in addition to improving the tooltip).